.eslintrc.json

```diff
@@ -5,7 +5,9 @@
         "node": true
     },
     "extends": [
-        "standard"
+        "standard",
+        "eslint:recommended",
```

Comment:
Why eslint-recommended…? Just asking because I thought it's already in there…?

Reply:
I saw it somewhere and thought "recommended sounds good". Couldn't really find a comparison of standard vs recommended but it seems that the standard style seems to be more strict in general. So I've removed recommended now again.